swap/swap.go

```diff
@@ -171,8 +172,10 @@ func (s *Swap) Add(amount int64, peer *protocols.Peer) (err error) {
 	// Check if balance with peer crosses the payment threshold
 	// It is the peer with a negative balance who sends a cheque, thus we check
 	// that the balance is *below* the threshold
-	if newBalance <= -s.paymentThreshold {
-		log.Warn("balance for peer went over the payment threshold, sending cheque", "peer", peer.ID().String(), "payment threshold", s.paymentThreshold)
+	paymentThreshold, _ := s.thresholdOracle.GetPaymentThreshold()
+
+	if newBalance <= -int64(paymentThreshold) {
```

Comment:
same here

Reply:
Not possible here, as the thresholdOracle returns two values. 
I think it is good, as is, as we are anyways comparing with the negative value of the `paymentThreshold` while the `log` says "went over", which is about the positive value of the payment threshold.

Reply:
ok, that makes sense. resolving